app/cdap/components/DataPrep/DataPrepSidePanel/index.js

```diff
@@ -140,6 +153,14 @@ export default class DataPrepSidePanel extends Component {
                 {T.translate(`${PREFIX}.targetTabLabel`)}
               </div>
             </If>
+            <If condition={this.state.showSampleColumn}>
```

Comment:
Ideally we don't want to use `If`. Use `{ condition && ...}`. But refactor can take lower priority

Reply:
Got it -- fixed!